### ISSUE TITLE
Flagg Model Tuning Plan

### ISSUE THREAD

**Issue by cflagg** [catullus/mm2017#7]
## Initial Model Results

* `Type` game by game 
* `Response` binary win/loss 
* `Training Set`: 2014-2017 regular season
* `Test Set`: 2014-2017 tournament results (take last game week of each tournament team as input)

* random forest (classification, object = `rf5wOA`) with 5 weeks of previous performance = 60% correct test predictions (76% OOB)
* ranger (probability tree, object = `ranger5wOA`) with 5 weeks of previous performance data = 51% correct test predictions (95% OOB, weird overfitting...)

## Path to Improve Predictions - game by game model

* Reduce input data? 
    * Maybe games from crap teams is confusing the model
    * Only include teams that have been in the tournament in any season?
* Reduce feature set to top 12 variables identified from random forest
* Try different previous performance windows
    * Last 3 games
    * Last 9 games
* Try reduced set with different models, but using same training and test data
    * Logistic Regression
    * Boosted Regression Tree
    * ANN
* Add simple variables already available:
    * Conference Name (factor/enum)

## End of Season Model

* `Model` Total Tournament Wins (in one season) ~ End of Season ("EoS") Stats
    * Where EoS = 
         * Average for identified stats
         * Variance for identified stats
         * Min/Max for identified stats

## Ensemble Predictions

* Make predictions on same test set
* Combine predictions across

**Comment by cflagg** [comment]
Also need to think about ROC/AUC curves and maximizing the true positive cut-off rate for binary classification models: https://stackoverflow.com/questions/16347507/obtaining-threshold-values-from-a-roc-curve

e.g. perhaps ideal cut-off for classifying a win/loss is not `>0.51` but perhaps `>0.65`
